Routers/userRouter.js

```diff
@@ -29,44 +31,51 @@ const router = express.Router();
           country: req.body.country,
           zip_code: req.body.zip_code,
           birthdate: req.body.birthdate,
-          date_registration: req.body.date_registration
+          date_registration: req.body.date_registration,
```

Comment:
jak poprawisz model to tutaj tez walidacja do poprawy nie dodawalismy tutaj ts'a wiec juz sie obejdziemy bez typowania tego jakiegos mocnego ale ogolnie jest taka mozliwosc wtedy nie trzeba byloby analizowac modelu zeby dobierac sie do odpowiednich rzeczy ale narazie to bez sensu utrudniac sobie zycie